### PR TITLE
LLM: Add qwen-vl gpu example

### PR DESCRIPTION
**Pull request by lalalapotter**
## Description

This PR is to add qwen-vl gpu example.

## How to test?
- [x] Application test

### CODE REVIEW

**Comment by Oscilloscope98** [comment]
We may need to change the example image

**Review by shane-huang — COMMENTED** [review]
you may use cocodataset image as reference input.

File: python/llm/example/GPU/HF-Transformers-AutoModels/Model/qwen-vl/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+-------------------- Session 4 --------------------
+ Please input a picture: exit
+```
+The sample input image in Session 1 is (which is fetched from [COCO dataset](https://cocodataset.org/#explore?id=264959)):
+
+<a href="http://farm6.staticflickr.com/5268/5602445367_3504763978_z.jpg"><img width=400px src="http://farm6.staticflickr.com/5268/5602445367_3504763978_z.jpg" ></a>
+
+The sample output image in Session 3 is:
+
+<a href="https://llm-assets.readthedocs.io/en/latest/_images/qwen-vl-example-output-gpu.png"><img width=250px src="https://llm-assets.readthedocs.io/en/latest/_images/qwen-vl-example-output-gpu.png" ></a>
```
Comment (Oscilloscope98) [inline python/llm/example/GPU/HF-Transformers-AutoModels/Model/qwen-vl/README.md:78]: width=400px

File: python/llm/example/GPU/HF-Transformers-AutoModels/Model/qwen-vl/chat.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    # Due to issue https://github.com/intel/intel-extension-for-pytorch/issues/454,
+    # currently put interpolation execution into cpu
+    def to_cpu(module, input, output):
+        return output.to("cpu")
+
+    def to_xpu(module, input):
+        return (input[0].to("xpu"),)
+
+    model.transformer.visual.ln_pre.register_forward_hook(to_cpu)
+    model.transformer.visual.transformer.register_forward_pre_hook(to_xpu)
```
Comment (Oscilloscope98) [inline python/llm/example/GPU/HF-Transformers-AutoModels/Model/qwen-vl/chat.py:56]: Please kindly add a blank line after line 56 to separate section :)

File: python/llm/example/GPU/PyTorch-Models/Model/qwen-vl/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+-------------------- Session 4 --------------------
+ Please input a picture: exit
+```
+The sample input image in Session 1 is (which is fetched from [COCO dataset](https://cocodataset.org/#explore?id=264959)):
+
+<a href="http://farm6.staticflickr.com/5268/5602445367_3504763978_z.jpg"><img width=400px src="http://farm6.staticflickr.com/5268/5602445367_3504763978_z.jpg" ></a>
+
+The sample output image in Session 3 is:
+
+<a href="https://llm-assets.readthedocs.io/en/latest/_images/qwen-vl-example-output-gpu.png"><img width=250px src="https://llm-assets.readthedocs.io/en/latest/_images/qwen-vl-example-output-gpu.png" ></a>
```
Comment (Oscilloscope98) [inline python/llm/example/GPU/PyTorch-Models/Model/qwen-vl/README.md:78]: width=400px

File: python/llm/example/GPU/PyTorch-Models/Model/qwen-vl/chat.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    # Due to issue https://github.com/intel/intel-extension-for-pytorch/issues/454,
+    # currently put interpolation execution into cpu
+    def to_cpu(module, input, output):
+        return output.to("cpu")
+
+    def to_xpu(module, input):
+        return (input[0].to("xpu"),)
+
+    model.transformer.visual.ln_pre.register_forward_hook(to_cpu)
+    model.transformer.visual.transformer.register_forward_pre_hook(to_xpu)
```
Comment (Oscilloscope98) [inline python/llm/example/GPU/PyTorch-Models/Model/qwen-vl/chat.py:58]: Same here :)

**Review by Oscilloscope98 — COMMENTED** [review]
Please also add the link [here](https://github.com/intel-analytics/BigDL/blob/main/README.md#verified-models)

**Comment by Oscilloscope98** [comment]
Please kindly resolve the conflicts :) 
We only keep the model table in BigDL/readme and BigDL/python/llm/readme for now

**Review by Oscilloscope98 — APPROVED** [review]
LGTM

**Pull request merged**